tests/unit/options.js

```diff
@@ -631,6 +631,39 @@ exports.safeasi = function (test) {
     .addError(10, 9, "Missing semicolon.")
     .test(src, { asi: true });
 
+  var afterBracket = [
+    'x = []',
+    '[1];',
+    'x[0]',
+    '(2);'
+  ];
+
+  TestRun(test, 'following bracket (asi: false)')
+    .test(afterBracket);
+
+  TestRun(test, 'following bracket (asi: true)')
+    .addError(2, 1, "Misleading line break before '['; readers may interpret this as an expression boundary.")
+    .addError(4, 1, "Misleading line break before '('; readers may interpret this as an expression boundary.")
+    .test(afterBracket, { asi: true });
+
+  var asClause = [
+    'if (true)',
+    '  ({x} = {});',
+    'if (true)',
+    '  [x] = [0];',
+    'while (false)',
+    '  ({x} = {});',
+    'while (fale)',
```

Comment:
I don't think it actually matters, but I figure I should at least call out this typo—in case you want to fix it?

Reply:
It probably doesn't matter, but I do want to fix it. Nice eye!